### DISCUSSION TITLE
Showcase are broken

### DISCUSSION THREAD

**Discussion by JulioJu** [pmndrs/jotai#1889] [Q&A] [answered]
Hi,

Some shocases are broken .

Follow README.md of 

* https://github.com/pmndrs/jotai/tree/1313a9d2fd07689e30a771ad5516d3de0e91f0a8/examples/hacker_news
* https://github.com/pmndrs/jotai/tree/1313a9d2fd07689e30a771ad5516d3de0e91f0a8/examples/todos 

We have error in browser 

```
Failed to compile

./src/App.tsx
Attempted import error: 'useSetAtom' is not exported from 'jotai'.

This error occurred during the build time and cannot be dismissed.
```

When we download this sample from https://jotai.org/docs/basics/showcase we have the same problem. Same problem with Yarn and Npm. 

I notice that is Jotai 1.x that is used in this examples.

Can I open an issue into Jotai ?

**Comment by dai-shi** [comment]
Nice catch! You can directly open a PR.
Thanks for your help!

**Reply by JulioJu** [reply, marked as answer]
Thanks for your answer @dai-shi .

I don't use Jotai, and today I've no time to investigate. I've created a new issue at https://github.com/pmndrs/jotai/issues/1890 .
